src/cli/command.py

```diff
@@ -0,0 +1,58 @@
+import re
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Any, List, Optional, Pattern
+
+from attr import dataclass
+from typing_extensions import Literal
+
+InputAllowedType = Literal["str", "directory", "path", "bool", "regexp"]
+
+
+class Command(ABC):
+    @dataclass
+    class Argument:
```

Comment:
Please make those two frozen

Reply:
I've made the `Argument` class Frozen. I'll leave `Command` class as it is because it doesn't have attributes.